superbench/benchmarks/micro_benchmarks/cublas_test.py

```diff
@@ -0,0 +1,307 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the cublas functions benchmarks."""
+
+import os
+import json
+import yaml
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry
+from superbench.benchmarks.micro_benchmarks import MicroBenchmarkWithInvoke
+
+
+class CublasBenchmark(MicroBenchmarkWithInvoke):
+    """The CublasBenchmark overhead benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+        self.default_params_dict_list = [
```

Comment:
private member, can use self.__default_params_dict_list

Reply:
I agree.

Reply:
ok, thank you for the comment